quarkchain/cluster/tests/test_cluster.py

```diff
@@ -43,7 +47,21 @@ def _tip_gen(shard_state):
     return b
 
 
+class NormalServer(grpc_pb2_grpc.ClusterSlaveServicer):
+    def SetRootChainConfirmedBlock(self, request, context):
+        unittest.TestCase.assertTrue(self, expr=request.message)
```

Comment:
you can do the following

```
class NormalServer()..

  def __init__(self):
    self.request_num = 0
```

then increment the counter in the handling method. then confirm the request number is correct